node-util/sbin/oo-admin-cartridge

```diff
@@ -124,7 +124,7 @@ parser  = OptionParser.new do |opts|
   opts.on('-o', '--cartridge_vendor VENDOR', 'Cartridge vendor of cartridge to erase. default: redhat') { |o| options.cartridge_vendor = o }
   opts.on('-n', '--name NAME', 'Name of cartridge to erase') { |o| options.name = o }
   opts.on('-v', '--version VERSION', 'Version of packaged software to erase') { |o| options.version = o }
-  opts.on('-c', '--cartridge_version VERSION', 'Version of cartridge to erase') { |o| options.cartridge_version = o }
+  opts.on('-c', '--cartridge_version VERSION', 'Version of cartridge to erase (Note: All software versions of given cartridge version will be erased.)') { |o| options.cartridge_version = o }
```

Comment:
Shouldn't this not be on the `-v` option?

Also, since all software versions will be deleted, would it be better to modify the `oo-admin-cartridge -c erase` logic to not require both the `-c` and `-v` options?